contribs/gmf/src/filters/filterselectorComponent.js

```diff
@@ -248,7 +248,7 @@ class Controller {
     /**
      * The data source ready to be filtered, after it has been selected and
      * prepared.
-     * @type {?import("gmf/datasource/OGC.js").default}
+     * @type {?import("gmf/datasource/OGC.js").OGCOptions}
```

Comment:
This is invalid.  The type of this property is not a hash of option. It's an instance of a class.

Reply:
Right, fixed be removing one @extent :-), Thanks.